src/core/Akka.NodeTestRunner/Program.cs

```diff
@@ -17,15 +18,22 @@ class Program
         static void Main(string[] args)
         {
             var nodeIndex = CommandLine.GetInt32("multinode.index");
-            var assemblyName = CommandLine.GetProperty("multinode.test-assembly");
+            var assemblyFileName = CommandLine.GetProperty("multinode.test-assembly");
```

Comment:
we receive the full path of the assembly we're testing as a constructor argument now, rather than just the name of the DLL. In fact, that's the way XUnit already works.